README.md

```diff
@@ -9,4 +9,13 @@ More information on the [about page](http://elite.laulhere.com/ExTool/index.php?
 * Download the [latest release](https://github.com/ExTool/ExTool-EDMC-Plugin/releases/latest).
 * Open the `.zip` archive that you downloaded and move the `ExTool` folder contained inside into the `plugins` folder.
 
-You will need to re-start EDMC for it to notice the new plugin.
\ No newline at end of file
+You will need to re-start EDMC for it to notice the new plugin.
+
+## EDMCOverlay
+
+ExTool has support for the [EDMCOverlay](https://github.com/inorton/EDMCOverlay) plugin. 
+If the [EDMCOverlay](https://github.com/inorton/EDMCOverlay) plugin is installed then the details of the current destination are displayed in the HUD
```

Comment:
I've actually not been maintaining releases of EDMCOverlay, it gets bundeled with HITS which has an installer and installs EDMCOverlay correctly